quickstarts/aws/aws-route-53/config.yml

```diff
@@ -1,38 +0,0 @@
-id: bf043e5a-213c-411e-8eea-4e386a29a088
```

Comment:
Is this quickstart being intentionally deleted? I didn't see a replacement, so I wanted to check.

Reply:
Thanks for spotting this! We are moving this one to another group for validation and I should have replaced it.